package.json

```diff
@@ -9,8 +9,9 @@
   "main": "src/main.ts",
   "scripts": {
     "build": "gulp",
-    "test": "exit 0",
+    "test": "NODE_ENV=testing mocha --exit -r ts-node/register --project tsconfig.json test/**/*.spec.ts",
```

Comment:
NODE_CONFIG_ENV (I think) should be testing and NODE_ENV as production.

This instructs node to compile libs in production mode but use testing config.